### PR TITLE
Change order v2 improvements

### PR DESCRIPTION
**Pull request by Isabella-Veloso**
Mostly added links to endpoint's descriptions.

#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

### CODE REVIEW

File: VTEX - Orders API.json [hunk 1]
```diff
@@ -7180,11 +7180,11 @@
...
...| \r\n| Checkout Admin | Orders Full Access |\r\n| OMS - Full access | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| IntegrationProfile - Fulfillment Oms | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| IntegrationProfile - Fulfillment Gateway | Cancel order |\r\n| IntegrationProfile - Fulfillment Gateway Oms | <p>Cancel order<\/p><p>Change order<\/p> |\r\n| inStore Sales Person | <p>Cancel order<\/p><p>Change order<\/p> |\r\n\r\n>\u2757 Assigning a [predefined role](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy) to users or application keys usually grants permission to multiple [License Manager resources](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3). If some of these permissions are not necessary, consider creating a custom role instead. For more information regarding security, see [Best practices for using application keys](https:\/\/help.vtex.com\/en\/tutorial\/best-practices-application-keys--7b6nD1VMHa49aI5brlOvJm).\r\n\r\nTo learn more about machine authentication at VTEX, see [Authentication overview](https:\/\/developers.vtex.com\/docs\/guides\/authentication-overview#machine-authentication).",
```
Comment (ricardoaerobr) [inline VTEX - Orders API.json:7187]: Em _u2757_  não seria o caso de utilizar ❗ ?

Comment (ricardoaerobr) [inline VTEX - Orders API.json:7187]: Em u26A0\uFE0F não seria o caso de utilizar ⚠️ ?

**Review by ricardoaerobr — CHANGES_REQUESTED** [review]
Verificar os comentários de callout.

File: VTEX - Orders API.json (hunk 1 above)
Comment (Isabella-Veloso) [reply VTEX - Orders API.json:7187]: Alterado.

Comment (Isabella-Veloso) [reply VTEX - Orders API.json:7187]: ALterado.

**Review by ricardoaerobr — APPROVED** [review]
LGTM!

**Pull request merged**